content/hi/docs/reference/glossary/qos-class.md

```diff
@@ -0,0 +1,21 @@
+---
+title: QoS Class
```

Comment:
Please check other localisations & see if this has been translated/transliterated to their respective language. Kindly do the same with this, as well.

Reply:
QoS is termed as QoS in many languages, only class has been translated.